### PR TITLE
Expose gltf2 eight weights.

### LINKED ISSUE #28840: 8 Bone Influence Animations do not function properly with glTF2 and ESCN
**Godot version:**
Tried with nightly ecceabe and 3.1-stable

**Issue description:**
When exporting from blender to Godot, a number of issues appear:
### Blender
https://gfycat.com/VariableCircularBighornsheep

### Exported escn
https://gfycat.com/LikelyGlamorousAlleycat

### Exported gltf
https://gfycat.com/WickedGargantuanHare

I'm not sure if it's an importer problem, exporter problem, or both. Better collada seems to work.

**Steps to reproduce:**
Import the models given in the zip file and play the animations
[3dreport.zip](https://github.com/godotengine/godot/files/3170266/3dreport.zip)

### PR DESCRIPTION
**Pull request by fire**
Support eight bone weights in glTF2.

Fixes: https://github.com/godotengine/godot/issues/28840

### CODE REVIEW

**Comment by fire** [comment]
Got a crash.

File: servers/rendering_server.cpp [hunk 1]
```diff
@@ -865,8 +865,22 @@ Error RenderingServer::mesh_create_surface_data_from_arrays(SurfaceData *r_surfa
 					ERR_FAIL_V(ERR_INVALID_DATA);
 				} break;
 			}
-
 			ERR_FAIL_COND_V(array_len == 0, ERR_INVALID_DATA);
+		} else if (i == RS::ARRAY_BONES) {
+			switch (p_arrays[i].get_type()) {
+				case Variant::PACKED_INT32_ARRAY: {
+					Vector<Vector3> v3 = p_arrays[RS::ARRAY_VERTEX];
+					Vector<float> bones = p_arrays[i];
```
Comment (reduz) [inline servers/rendering_server.cpp:873]: probably meant to use Vector<uint32_t> here

File: servers/rendering_server.cpp [hunk 2]
```diff
@@ -865,8 +865,22 @@ Error RenderingServer::mesh_create_surface_data_from_arrays(SurfaceData *r_surfa
...
+					int32_t bone_group_count = bones.size();
+					int32_t vertex_count = v3.size();
+					if (vertex_count == bone_group_count / (ARRAY_WEIGHTS_SIZE + ARRAY_WEIGHTS_SIZE)) {
```
Comment (reduz) [inline servers/rendering_server.cpp:876]: I think ARRAY_WEIGHT_SIZE * 2 is more readable, but guess up to you

File: servers/rendering_server.cpp (hunk 1 above)
Comment (fire) [reply servers/rendering_server.cpp]: Changed to Vector<int32> as used in other places for RS::ARRAY_BONES

**Review by reduz — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**